README.md

```diff
@@ -157,6 +164,7 @@ Commands:
   cli.js generate <command>  Generate <aux>
   cli.js get <command>       Get an entity <assignment>
```

Comment:
This should state `cli.js get <id>`. The description should say "Get assignment given by <id>". Generally, the convention in UNIX help is that things that are within "<>" are variables holding user input and that descriptions only reference things that are part of the command.